packages/flutter/lib/src/gestures/converter.dart

```diff
@@ -31,6 +31,7 @@ class _PointerState {
   Offset lastPosition;
 }
 
+// ignore: avoid_classes_with_only_static_members
```

Comment:
Without this ignore we'd need to break the API and to find a good top level name for `PointerEventConverter.expand` (`expand` is too general imho).